crates/rnote-engine/src/pens/brush.rs

```diff
@@ -229,6 +230,28 @@ impl PenBehaviour for Brush {
                             );
                         }
 
+                        // remove strokes that follow a selection cancellation if they are large
+                        // hence we can write after selecting strokes but we won't leave tiny spots
+                        // behind
+                        let volume = engine_view
+                            .store
+                            .get_stroke_ref(*current_stroke_key)
+                            .unwrap()
+                            .bounds()
+                            .volume();
+                        if engine_view.store.get_cancelled_state()
+                            && volume
+                                < 4.0
```

Comment:
What about naming this value by adding a constant variable to make it more readable?

Reply:
Yes, done. I've added a little docstring to it as well